Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -436,6 +448,8 @@ - (void)_installScreenTimeWebpageControllerIfNeeded
         [screenTimeView setFrame:self.bounds];
         [self addSubview:screenTimeView.get()];
     }
+
+    _screenTimeWebpageControllerViewInstalled = YES;
```

Comment:
I don't think this variable is needed, because `viewIfLoaded` should give you the same information.